packages/angular/src/stories/all.stories.ts

```diff
@@ -64,10 +64,14 @@ storybookDemoGroups.forEach((demoGroup: DemoGroup) => {
 	// Create story group
 	const groupStories = storiesOf(`${demoGroup.storyGroupTitle}/${demoGroup.title}`, module)
 
+	groupStories.addDecorator(story => {
+		document.documentElement.setAttribute('data-carbon-theme', DEFAULT_THEME)
+		return story()
+	})
+
 	// Create stories within story group
 	demoGroup.demos.forEach((demo: Demo) => {
 		demo.options.theme = DEFAULT_THEME
-		document.documentElement.setAttribute('data-carbon-theme', DEFAULT_THEME)
```

Comment:
Bug because the theme was set once when storybook starts rather than each time an individual story loads. If you have some stories with the white theme and others g100, it doesn't work well this way.